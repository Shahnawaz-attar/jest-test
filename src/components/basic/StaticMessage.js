import React from 'react'

const StaticMessage = () => {
  return (
    <div>This is a static dffghf.</div>
  )
}

export default StaticMessage