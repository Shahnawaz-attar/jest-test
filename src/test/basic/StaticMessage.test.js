import "@testing-library/jest-dom"
import {render} from "@testing-library/react"
import StaticMessage from "../../components/basic/StaticMessage"



describe('StaticMessage Component',()=>{

    test('matches snapshot', () =>{

        const {asFragment} = render(<StaticMessage/>)
        expect(asFragment()).toMatchSnapshot()
    })
})
