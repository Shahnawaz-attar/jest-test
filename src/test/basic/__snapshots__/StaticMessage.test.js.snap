// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StaticMessage Component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    This is a static dffghf.
  </div>
</DocumentFragment>
`;
